docs/getting-started/first-traffic-split.md

```diff
@@ -142,7 +143,7 @@ Verify traffic split configure correctly by listing the revisions again.
 ```{ .bash .no-copy }
 NAME            SERVICE   TRAFFIC   TAGS   GENERATION   AGE   CONDITIONS   READY   REASON
 hello-knative   hello     50%              2            10m   3 OK / 4     True
-hello-world     hello     50%              1            36m   3 OK / 4     True 
+hello-world     hello     50%              1            36m   3 OK / 4     True
```

Comment:
If you name your revisions `world` and `knative` then remove `hello-` from the column `NAME` the column `SERVICE` containers `hello`

Also you need to make the CLI commands match to be consistence for example:
Instead of using revision `hello-world`
```
kn service update hello  --traffic hello-world=50  --traffic @latest=50
```
Use revision name `world`
```
kn service update hello  --traffic world=50   --traffic @latest=50
```

Reply:
I double check if revision is `knative` the name will always have the service name prefix